### ISSUE TITLE
Relic probability density seems incorrect

### ISSUE THREAD

**Issue by eugenelourie** [Lux-AI-Challenge/Lux-Design-S3#44]
Hello! I might be mistaking, but these lines look strange:
https://github.com/Lux-AI-Challenge/Lux-Design-S3/blob/71236024b06fb65118dc814b483339e8c7ebc11f/src/luxai_s3/state.py#L279C9-L292C10

Comment in the line 279 states that the probability density is 25%, but after that the `randint` function is used:
`relic_node_configs = (
    jax.random.randint(...
        maxval=10,
    ).astype(jnp.float32)
    >= 7.5
)`

Shouldn't it be something like:
`relic_node_configs = (
    jax.random.randint(...
        maxval=100,
    ).astype(jnp.float32)
    >= 75
)`

**Comment by StoneT2000** [comment]
Yes technically its not 25%, i'll update the comment, leave this in as a feature for now :)
